multi-region-guardduty/variables.tf

```diff
@@ -0,0 +1,35 @@
+# ------------------------------------------------------------------------------
+# Variables
+# ------------------------------------------------------------------------------
+variable "name_prefix" {
+  description = "A prefix used for naming resources."
+  default     = ""
+}
+
+variable "regions" {
+  description = "A list of regions in which to enable GuardDuty."
+
+  default = [
+    "us-east-1",
```

Comment:
Would be nice if this list was in the same order as the provider blocks in the regions.tf file